tests/e2e/specs/in-game/spectating/live-spectating.spec.js

```diff
@@ -30,10 +30,26 @@ describe('Spectating Games', () => {
 
   it('Prevents spectating your own game while it\'s ongoing', () => {
     cy.setupGameAsP0();
+    cy.loadGameFixture(0, {
+      p0Hand: [ Card.ACE_OF_SPADES, Card.ACE_OF_CLUBS ],
+      p0Points: [ Card.TEN_OF_SPADES ],
+      p0FaceCards: [ Card.KING_OF_SPADES ],
+      p1Hand: [ Card.ACE_OF_HEARTS, Card.ACE_OF_DIAMONDS, Card.EIGHT_OF_DIAMONDS ],
+      p1Points: [ Card.TEN_OF_HEARTS ],
+      p1FaceCards: [ Card.KING_OF_HEARTS ],
+    });
+
+    assertGameState(0, {
+      p0Hand: [ Card.ACE_OF_SPADES, Card.ACE_OF_CLUBS ],
+      p0Points: [ Card.TEN_OF_SPADES ],
+      p0FaceCards: [ Card.KING_OF_SPADES ],
+      p1Hand: [ Card.ACE_OF_HEARTS, Card.ACE_OF_DIAMONDS, Card.EIGHT_OF_DIAMONDS ],
+      p1Points: [ Card.TEN_OF_HEARTS ],
+      p1FaceCards: [ Card.KING_OF_HEARTS ],
+    });
     cy.get('@gameId').then((gameId) => {
       cy.visit(`/spectate/${gameId}`);
-      cy.url().should('not.include', `/spectate/${gameId}`);
-      assertSnackbar(`Cannot spectate game because you are playing`, 'error', 'newgame');
```

Comment:
Instead of going home and getting the snackbar you now go to the game without an error